lib/p2p/Pool.ts

```diff
@@ -177,7 +177,6 @@ class Pool extends EventEmitter {
       throw err;
     } else if (this.peers.has(nodePubKey)) {
       const err = errors.NODE_ALREADY_CONNECTED(nodePubKey, address.host);
-      this.logger.warn(err.message);
```

Comment:
Is there a reason you dropped this message? Is something similar being logged elsewhere? Just wondering.